docs/scripting-ref.rst

```diff
@@ -2230,3 +2230,30 @@ Trigger mouse release event in web page.
 **Async:**: no.
 
 See notes about mouse events in :ref:`splash-mouse-click`.
+
+
+.. _splash-with-timeout:
+
+splash:with_timeout
+-------------------
+
+Run the callback with the allowed timeout
+
+**Signature:** ``ok, result = splash:with_timeout(callback, timeout)``
```

Comment:
Yeah, it is easier for my eyes as well. But there are some stupid tech difficulties in having timeout first ([sets_callback](https://github.com/scrapinghub/splash/blob/ad656e3ee22f7b3161279777cb7cc0d12a47d786/splash/lua_modules/wraputils.lua#L100) uses first argument), so for `call_later` I made timeout the last argument. It is also consistent with  JavaScript functions. In Python it is more common to have timeout a first argument, e.g. this is the case in asyncio and twisted.